content/remote-development/init-command.md

```diff
@@ -3,17 +3,21 @@ title: Init Command
 menu:
   main:
     parent: 'remote-development'
-    weight: 90
+    weight: 20
 
-weight: 90
+weight: 20
 ---
 ## Using the Init Command
 
 ```
 cp-remote init [token]
 ```
 
-The `init` command will initialise a remote environment using a token generated by the Continuous Pipe UI. 
+The `init` command will initialise a remote environment using a token generated by the [ContinuousPipe console](https://ui.continuouspipe.io/). 
```

Comment:
Reading this sentence again, it could cause confusion as its not clear that the token needs generating separately. Might be better to say something like "The `init` command will initialise a remote environment using an authorisation token. Tokens can be generated in the [ContinuousPipe console](https://ui.continuouspipe.io/)."